### ISSUE TITLE
Container release

### ISSUE THREAD

**Issue by phoenix-bjoern** [LINBIT/linstor-gui#2]
First of all: Thank you very much for open sourcing this useful tool! I guess many admins very much appreciate this big step :-)

Linbit SDS is often used in a container context (e.g. Kubernetes). Therefore it would be useful if you could publish container images on every release. The Makefile already contains the necessary build step, probably only the GH action is missing to execute and publish the images on a new release. I would propose to use the existing Quay account (https://quay.io/organization/linbit).

**Comment by phoenix-bjoern** [comment]
@rck I disagree. The GUI is a web application. The proper approach to host it in K8S is to have an ingress (domain, SSL, authentication), a service and a deployment (resource control/limits). The requirements for running the GUI is completely different from running the controller in K8S. Sure you can run the controller and the GUI image side-by-side in the same namespace, but maybe you also want to run it in a different cluster and just allow communication with the controller.
So, IMHO having a container release of the GUI is a must have. Please don't mix it up with the Linstor controller.

**Comment by Liang0028** [comment]
An experimental Docker deployment solution has been implemented and merged https://github.com/LINBIT/linstor-gui/commit/65df17320009138179dee74a1f7c9c19b390ba8e into the main branch. If you're interested, we encourage you to test it out.

**Comment by Liang0028** [comment]
> Hi,
> 
> First of all: Thank you very much for open sourcing linstor-gui
> 
> Your experimental Docker deployment solution works as expected 👍
> 
> We have submitted some improvements to you including native https support see pr #7

Thank you very much. However, our release process has not yet been migrated to GitHub, so we may need to address that first. I will discuss this proposal with the relevant people. Thanks again!

**Comment by boedy** [comment]
Having https://hub.docker.com/u/linbit as a verified trusted source to pull from would also be a good option.

**Comment by phoenix-bjoern** [comment]
@WanzenBug Wait, you just added it to the controller in [this commit](https://github.com/piraeusdatastore/piraeus/commit/8f2ccda3554091093337b9b5a63587d19764fea0)? LOL ;-) 
While I really appreciate your initiative, IMHO the Linstor Controller container shouldn't be mixed up with the GUI. Adding the GUI in the container doesn't just add 8 MB of JavaScript, there also has to be web server to expose the GUI.
K8S gives the option to combine several containers to a pod, so you can use ports or even sockets to allow communication between the controller and the GUI. "Separation of concerns" should be respected, especially in this case where it is also necessary to start the GUI in the container and add a service (and maybe an ingress).
I always appreciate your work, but I disagree that adding the GUI (files) to the controller container is a good idea as it requires more to run them. Also adding dependencies to the controller container isn't good in regards to the release cycles.
